lists/global.csv

```diff
@@ -1144,7 +1144,7 @@ https://protonmail.com/,HACK,Hacking Tools,2018-03-19,OONI,
 https://getoutline.org/,ANON,Anonymization and circumvention tools,2018-04-06,Duplika,
 http://fteproxy.org/,ANON,Anonymization and circumvention tools,2018-05-21,Community member,
 https://www.thegeekdiary.com/,HACK,Hacking Tools,2018-05-21,Community member,
-https://dns.google.com/,HOST,Hosting and Blogging Platforms,2018-05-02,OONI,
+https://dns.google.com/resolve?name=www.example.com&type=A,HOST,Hosting and Blogging Platforms,2018-05-02,OONI,
```

Comment:
I would say that, in light of how the web connectivity test checks for consistency of sites, since you are doing an encrypted HTTPS request we are more likely to accurately detect consistencies/inconsistencies with the index page which returns HTML content vs the JSON payload.

My suggestion is to leave this as is, and if it's useful to have a query, to add it as another row